api-report/container-definitions.api.md

```diff
@@ -444,7 +444,7 @@ export interface IResolvedFluidCodeDetails extends IFluidCodeDetails {
 export interface IRuntime extends IDisposable {
     createSummary(blobRedirectTable?: Map<string, string>): ISummaryTree;
     getEntryPoint?(): Promise<FluidObject | undefined>;
-    getPendingLocalState(): unknown;
+    getPendingLocalState(close?: boolean): unknown;
```

Comment:
can we make this an object, rather than a single param, so its a bit more furture proof:
```suggestion
    getPendingLocalState(props?:{close?: boolean}): unknown;
```

Reply:
also, we should try to come up with a better name than close, something that more accurately describes the behavior.